Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -1381,7 +1381,9 @@
 ;; <rdar://problem/47268166>
 (allow mach-lookup
     (require-all
+#if !PLATFORM(MAC) || __MAC_OS_X_VERSION_MIN_REQUIRED >= 130000
         (extension "com.apple.webkit.extension.mach")
```

Comment:
With DOM rendering off, I believe this extension should be present, which should allow the mach-lookup.

Reply:
I see! Something else may be causing this extension to be intermittently missing when using multiple tabs.

Reply:
Since DOM rendering is off by default on __MAC_OS_X_VERSION_MIN_REQUIRED < 14, I think this patch looks good.

Perhaps it could be changed to `#if PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 140000`, since Catalyst seems to not be affected by this?

Reply:
Sounds good, updated PR to use `#if PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 140000`.

Reply:
I looks like it is not accepted to have a single condition inside `require-all`, so a slight rewrite might be needed.

Reply:
Thanks! To avoid changing for Catalyst, should this be `!PLATFORM(MAC) || __MAC_OS_X_VERSION_MIN_REQUIRED >= 140000` now?